src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -3,9 +3,23 @@ import { useUser } from '../../context/UserContext';
 
 // A wrapper for <Route> that redirects to the login
 // screen if you're not yet authenticated.
-export default function PrivateRoute({ children, ...rest }) {
+export default function PrivateRoute({ children, ...routeProps }) {
   // TODO: Use the user in context to determine whether to redirect to /login
-  // const { user } = useUser();
+  // useUser must be used within UserProvider
+  const { user } = useUser();
 
-  return <Route {...rest} render={({ location }) => children} />;
```

Comment:
Nice work finishing this here creating a private route that takes in the children and router props and determines where to send the user if they are signed in